src/validation.h

```diff
@@ -857,11 +857,15 @@ class ChainstateManager
     /**
      * If a block header hasn't already been seen, call CheckBlockHeader on it, ensure
      * that it doesn't descend from an invalid block, and then add it to m_block_index.
+     * Caller must set min_pow_checked=true in order to add a new header to the
+     * block index (permanent memory storage), indicating that the header is
+     * known to be part of a sufficiently high-work chain (anti-dos check).
      */
     bool AcceptBlockHeader(
         const CBlockHeader& block,
         BlockValidationState& state,
-        CBlockIndex** ppindex) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
+        CBlockIndex** ppindex,
+        const bool min_pow_checked) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
```

Comment:
In commit "Require callers of AcceptBlockHeader() to perform anti-dos checks"

Here too, the `const` is meaningless.

Reply:
Removed.